_events/2020-01-20-GalaxyWS_scrna_FR.md

```diff
@@ -0,0 +1,125 @@
+---
+site: freiburg
+tags:
+- training
+title: Galaxy single cell RNA-seq data analysis workshop in Freiburg
+starts: 2020-03-23
+ends: 2020-03-27
+organiser:
+  name: Freiburg Galaxy Team & MPI Freiburg
+  email: contact@usegalaxy.eu
+location:
+  name: PC Pool 3
+  street: Werthmannstraße 4 
+  postal: 79098
+  city: Freiburg im Breisgau
+  region: Baden-Württemberg
+  country: Germany
+  geo:
+    lat: 47.993469
+    lon: 7.845275
+supporters:
+- unifreiburg
+- denbi
+- mpi
```

Comment:
if MPI is not involved, remove

Reply:
should I keep deNBI?

Reply:
SURE! Most important. You could add elixir as well because denbi is the German elixir node